src/Sylius/Behat/Context/Api/Shop/ProductVariantContext.php

```diff
@@ -17,25 +17,27 @@
 use Sylius\Behat\Client\ApiClientInterface;
 use Sylius\Behat\Client\ResponseCheckerInterface;
 use Sylius\Behat\Service\SharedStorageInterface;
-use Sylius\Component\Core\Formatter\StringInflector;
-use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariantInterface;
 use Webmozart\Assert\Assert;
 
 final class ProductVariantContext implements Context
 {
     private ApiClientInterface $client;
```

Comment:
it just my opinion but when we have more than a single client in context we should name it more explicit (for example `variantClient`